### PR TITLE
refactor: ORG instruction

### PR DESCRIPTION
**Pull request by slavek-kucera**

### CODE REVIEW

**Comment by slavek-kucera** [comment]
Originally, I wanted to do something more significant, but the impact is hard to measure with the other sources of overhead...

File: parser_library/src/context/ordinary_assembly/symbol_dependency_tables.cpp [hunk 1]
```diff
@@ -62,15 +62,15 @@ struct resolve_dependant_visitor
 {
     symbol_value& val;
     loctr_dependency_resolver* resolver;
-    ordinary_assembly_context& sym_ctx_;
-    std::unordered_map<dependant, statement_ref>& dependency_source_stmts_;
+    ordinary_assembly_context& m_sym_ctx;
```
Comment (jirimosinger) [inline parser_library/src/context/ordinary_assembly/symbol_dependency_tables.cpp:65]: Consider adding a prefix `m_` to the rest of the variables as well

File: parser_library/src/context/ordinary_assembly/symbol_dependency_tables.cpp [hunk 2]
```diff
@@ -316,28 +313,28 @@ void symbol_dependency_tables::add_dependency(space_ptr target,
...
 
     add_dependency(dependant(target), &*it->second, false, dep_ctx);
 
     if (dependency_source_stmt)
     {
-        auto [sit, sinserted] = postponed_stmts_.try_emplace(std::move(dependency_source_stmt), dep_ctx);
+        auto [sit, sinserted] = m_postponed_stmts.try_emplace(std::move(dependency_source_stmt), dep_ctx);
 
         if (!sinserted)
             throw std::runtime_error("statement already registered");
```
Comment (jirimosinger) [inline parser_library/src/context/ordinary_assembly/symbol_dependency_tables.cpp:325]:
```suggestion
        assert (sinserted);
```

File: parser_library/src/context/ordinary_assembly/symbol_dependency_tables.cpp [hunk 3]
```diff
@@ -316,28 +313,28 @@ void symbol_dependency_tables::add_dependency(space_ptr target,
     const dependency_evaluation_context& dep_ctx,
     post_stmt_ptr dependency_source_stmt)
 {
-    auto [it, inserted] = dependency_source_addrs_.emplace(target, std::move(dependency_source));
+    auto [it, inserted] = m_dependency_source_addrs.emplace(target, std::move(dependency_source));
```
Comment (jirimosinger) [inline parser_library/src/context/ordinary_assembly/symbol_dependency_tables.cpp:317]:
```suggestion
    auto [it, inserted] = m_dependency_source_addrs.emplace(target, std::move(dependency_source));
    assert(inserted);
```

?

**Review by jirimosinger — APPROVED** [review]

**Review by jirimosinger — APPROVED** [review]

**Pull request merged**